ckanext/canada/plugins.py

```diff
@@ -402,6 +402,9 @@ def before_index(self, data_dict):
         if 'fgp_viewer' in data_dict.get('display_flags', []):
             data_dict['fgp_viewer'] = 'map_view'
 
+        titles = json.loads(data_dict.get('title_translated', '{}'))
+        data_dict['title_fr'] = titles.get('fr','')
+        data_dict['title'] = titles.get('en','')
```

Comment:
french is going into `title_fr`, english is going into `title` but it looks like we're sorting english on `title_string`